### PR TITLE
Add Nomic embeddings

### PR DESCRIPTION
**Pull request by kacperlukawski**
This PR adds Nomic embeddings to the list of supported providers. It contains a simple example of how to use the embeddings in Python.

References:
- https://blog.nomic.ai/posts/nomic-embed-text-v1
- https://docs.nomic.ai/reference/endpoints/nomic-embed-text

### CODE REVIEW

**Comment by kacperlukawski** [comment]
Preview: https://deploy-preview-594--condescending-goldwasser-91acf0.netlify.app/documentation/embeddings/nomic/

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Nomic"
+weight: 1100
+---
+
+# Nomic
+
+The `nomic-embed-text-v1` model is an [open source](https://github.com/nomic-ai/contrastors) 8192 context length text encoder.
```
Comment (mjang) [inline qdrant-landing/content/documentation/embeddings/nomic.md:8]: While we like open source, the link is to the Nomic contrastor. If I understand correctly:

```suggestion
The `nomic-embed-text-v1` model is an open source [8192 context length]((https://github.com/nomic-ai/contrastors)) text encoder.
```

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+It is available on [Hugging Face Hub](https://huggingface.co/nomic-ai/nomic-embed-text-v1), but the easiest way to use it is 
+through the [Nomic Text Embeddings](https://docs.nomic.ai/reference/endpoints/nomic-embed-text), either by using the official
+Python client or by sending HTTP requests directly.
```
Comment (mjang) [inline qdrant-landing/content/documentation/embeddings/nomic.md:11]: Unclear: what's the "official" Python client? (I'm guessing we should drop the word)

```suggestion
While you can find it on the [Hugging Face Hub](https://huggingface.co/nomic-ai/nomic-embed-text-v1), 
you may have more success through the [Nomic Text Embeddings](https://docs.nomic.ai/reference/endpoints/nomic-embed-text).
Once installed, you can configure it with the official Python client or through direct HTTP requests.
```

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Nomic embeddings might be used directly in the Qdrant client's calls. Please note there is a difference in the way the embeddings
+are obtained for the documents and queries. The `task_type` parameter defines what kind of embeddings you want to obtain.
+For the documents, it should be set to `search_document`:
```
Comment (mjang) [inline qdrant-landing/content/documentation/embeddings/nomic.md:15]:
```suggestion
You can use Nomic embeddings directly in Qdrant client calls. There is a difference in the way the embeddings
are obtained for documents and queries. The `task_type` parameter defines the embeddings that you get.
For documents, set the `task_type` to `search_document`:
```

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+qdrant_client.upsert(
+    collection_name="my-collection",
+    points=models.Batch(
+        ids=[1, 2],
+        vectors=output["embeddings"],
+    ),
+)
+```
+
+While querying the collection, the `task_type` should be set to `search_query`:
```
Comment (mjang) [inline qdrant-landing/content/documentation/embeddings/nomic.md:37]:
```suggestion
To query the collection, set the `task_type` to `search_query`:
```

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    task_type="search_query",
+)
+
+qdrant_client.search(
+    collection_name="my-collection",
+    query=output["embeddings"][0],
+)
+```
+
+Please refer to the [Nomic documentation](https://docs.nomic.ai/reference/endpoints/nomic-embed-text) for more details.
```
Comment (mjang) [inline qdrant-landing/content/documentation/embeddings/nomic.md:52]:
```suggestion
For more information, see the Nomic documentation on [Text embeddings](https://docs.nomic.ai/reference/endpoints/nomic-embed-text).
```

**Review by mjang — CHANGES_REQUESTED** [review]
Thanks for tagging me, @kacperlukawski ! 

Note: I did not set anything up to check the code samples.

I've suggested changes to help the reader think "I can do this". (Thus my use of the second person ["you"])

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Nomic"
+weight: 1100
+---
+
+# Nomic
+
+The `nomic-embed-text-v1` model is an [open source](https://github.com/nomic-ai/contrastors) 8192 context length text encoder.
+It is available on [Hugging Face Hub](https://huggingface.co/nomic-ai/nomic-embed-text-v1), but the easiest way to use it is
```
Comment (AndriyMulyar) [inline qdrant-landing/content/documentation/embeddings/nomic.md:9]:
```suggestion
You can access Nomic Embeddings through the [official Nomic inference API](https://docs.nomic.ai/reference/endpoints/nomic-embed-text) or download the model weights on the [Hugging Face Hub](https://huggingface.co/nomic-ai/nomic-embed-text-v1).
```

File: qdrant-landing/content/documentation/embeddings/nomic.md (hunk 2 above)
Comment (AndriyMulyar) [reply qdrant-landing/content/documentation/embeddings/nomic.md:11]: Added a suggestion above on wording to make it more concise.

**Review by AndriyMulyar — APPROVED** [review]
Approved with my wording suggestion

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+Nomic embeddings might be used directly in the Qdrant client's calls. Please note there is a difference in the way the embeddings
+are obtained for the documents and queries. The `task_type` parameter defines what kind of embeddings you want to obtain.
+For the documents, it should be set to `search_document`:
+
+```python
+from qdrant_client import QdrantClient, models
+from nomic import embed
+
+output = embed.text(
+    texts=["The best vector database"],
```
Comment (AndriyMulyar) [inline qdrant-landing/content/documentation/embeddings/nomic.md:22]:
```suggestion
    texts=["Qdrant is the best vector database!"],
```

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+        vectors=output["embeddings"],
+    ),
+)
+```
+
+While querying the collection, the `task_type` should be set to `search_query`:
+
+```python
+output = embed.text(
+    texts=["The best vector database"],
```
Comment (AndriyMulyar) [inline qdrant-landing/content/documentation/embeddings/nomic.md:41]:
```suggestion
    texts=["What is the best vector database?"],
```

**Review by AndriyMulyar — CHANGES_REQUESTED** [review]

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Nomic"
+weight: 1100
+---
+
+# Nomic
+
+The `nomic-embed-text-v1` model is an open source [8192 context length]((https://github.com/nomic-ai/contrastors)) text encoder.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/embeddings/nomic.md:8]:
```suggestion
The `nomic-embed-text-v1` model is an open source [8192 context length](https://github.com/nomic-ai/contrastors)) text encoder.
```

File: qdrant-landing/content/documentation/embeddings/nomic.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Nomic"
+weight: 1100
+---
+
+# Nomic
+
+The `nomic-embed-text-v1` model is an open source [8192 context length](https://github.com/nomic-ai/contrastors)) text encoder.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/embeddings/nomic.md:8]:
```suggestion
The `nomic-embed-text-v1` model is an open source [8192 context length](https://github.com/nomic-ai/contrastors) text encoder.
```

**Comment by kacperlukawski** [comment]
Thanks for the review, @mjang I'd appreciate your second look and hopefully we can merge it and announce on Monday ;)

**Review by mjang — APPROVED** [review]
Thanks!

**Pull request merged**